src/__tests__/themeContext.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { decodeBase64 } from '../helper/utils';
+import fs from 'fs-extra';
+import path from 'path';
+import { init } from '../fdk';
+import rimraf from 'rimraf';
+import { readFile } from '../helper/file.utils';
+import CommandError from '../lib/CommandError';
+
+jest.mock('inquirer');
+let program;
+
+afterEach(() => {
+    configStore.clear();
+});
+
+afterAll(() => {
+    let filePath = path.join(process.cwd(), './.fdk/context.json');
+    try {
+        rimraf.sync(filePath);
+    } catch (err) {
+        console.error(`Error while deleting ${filePath}.`);
+    }
+});
+async function login() {
+    const inquirerMock = mockFunction(inquirer.prompt);
+    inquirerMock.mockResolvedValue({ password: '1234567' });
+    await program.parseAsync(['ts-node', './src/fdk.ts', 'login', '-e', 'anuragpandey@gofynd.com']);
+}
+
+describe('Theme Context Commands', () => {
+    beforeEach(async () => {
+        await program.parseAsync(['ts-node', './src/fdk.ts', 'env', 'set', '-n', 'fyndx0']);
+        program.commands.forEach(command => {
+            command._optionValues = {};
+        });
+    });
+
+    beforeAll(async () => {
+        program = await init('fdk');
+        const mock = new MockAdapter(axios);
+        mock.onPost(`${URLS.LOGIN_USER()}`).reply(200, data, {
+            'set-cookie': [{ Name: 'Anurag Pandey' }],
+        });
+        mock.onGet(`${URLS.OAUTH_TOKEN(appConfig.company_id)}`).reply(200, oauthData);
+        mock.onGet(
+            `${URLS.GET_APPLICATION_DETAILS(appConfig.application_id, appConfig.company_id)}`
+        ).reply(200, appConfig);
+        mock.onGet(
+            `${URLS.THEME_BY_ID(appConfig.application_id, appConfig.company_id, appConfig.theme_id)}`
+        ).reply(200, appConfig);
+    });
+
+    it('should successfully add theme context ', async () => {
+        await login();
+        const inquirerMock = mockFunction(inquirer.prompt);
+        inquirerMock.mockResolvedValue({ showCreateFolder: 'Yes' });
+        await program.parseAsync([
+            'ts-node',
+            './src/fdk.ts',
+            'theme',
+            'context',
+            '-t',
+            'eyJhcHBsaWNhdGlvbl9pZCI6IjYyMjg5NDY1OWJhYWNhM2JlODhjOWQ2NSIsInRva2VuIjoiNEVvaC15RE1XIiwiY29tcGFueV9pZCI6MSwiZXhwaXJlc19pbiI6IjIwMjItMDUtMDlUMDQ6NDg6MjguODg5WiIsInRoZW1lX2lkIjoiNjIzYjA5ZmFlYjBiNmUwZjRmZjk3NThmIn0=',
+            '-n',
+            'fyndx0',
+        ]);
+      let context: any = readFile(path.join(process.cwd(),'./.fdk/context.json'))
+            try {
+                context = JSON.parse(context);
+            } catch (e) {
+                throw new CommandError(`Invalid config.json`);
+            }
+        const configObj = JSON.parse(
+            decodeBase64(
+                'eyJhcHBsaWNhdGlvbl9pZCI6IjYyMjg5NDY1OWJhYWNhM2JlODhjOWQ2NSIsInRva2VuIjoiNEVvaC15RE1XIiwiY29tcGFueV9pZCI6MSwiZXhwaXJlc19pbiI6IjIwMjItMDUtMDlUMDQ6NDg6MjguODg5WiIsInRoZW1lX2lkIjoiNjIzYjA5ZmFlYjBiNmUwZjRmZjk3NThmIn0='
```

Comment:
add tokens two tokens one for init and one for create a theme in one common file import and use it everywhere @anuragpandey1115